### ISSUE TITLE
Exact paths with colons cannot be routed to correctly

### ISSUE THREAD

**Issue by jrhee17** [line/armeria#4577]
We have received a report that it isn't possible to add a service with a colon included.

Although the following path pattern is allowed:

```
sb.service("exact:/resource/name:customVerb", (ctx, req) -> HttpResponse.of(200));
```

a 404 is returned because the colon is mistakenly interpreted as a parameterized route

https://github.com/line/armeria/blob/de1533a73c4815f7f7eab16d9554f2a188ede993/core/src/main/java/com/linecorp/armeria/server/RoutingTrieBuilder.java#L198

Ref: https://stackoverflow.com/questions/74815759/how-to-include-in-an-http-path-in-an-armeria-server

**Comment by ikhoon** [comment]
It is a known bug. the problem will be fixed if #4064 is implemented.

**Comment by ikhoon** [comment]
It is time to gently ping @hyangtack. 😉

**Comment by minwoox** [comment]
Closed by #5676
